File: src/main/scala/grp/faq/entity/Message.java
package grp.faq.entity;

public class Message {
    private Integer cMessageId;

    private String cMessageSource;

    private Integer cMessageType;

    private Integer cReaded;

    public Message(Integer cMessageId, String cMessageSource, Integer cMessageType, Integer cReaded) {
        this.cMessageId = cMessageId;
        this.cMessageSource = cMessageSource;
        this.cMessageType = cMessageType;
        this.cReaded = cReaded;
    }

    public Message() {
        super();
    }

    public Integer getcMessageId() {
        return cMessageId;
    }

    public void setcMessageId(Integer cMessageId) {
        this.cMessageId = cMessageId;
    }

    public String getcMessageSource() {
        return cMessageSource;
    }

    public void setcMessageSource(String cMessageSource) {
        this.cMessageSource = cMessageSource == null ? null : cMessageSource.trim();
    }

    public Integer getcMessageType() {
        return cMessageType;
    }

    public void setcMessageType(Integer cMessageType) {
        this.cMessageType = cMessageType;
    }

    public Integer getcReaded() {
        return cReaded;
    }

    public void setcReaded(Integer cReaded) {
        this.cReaded = cReaded;
    }
}

File: src/main/scala/grp/faq/entity/Message.scala
package grp.faq.entity

import scala.beans.BeanProperty

class Message() {
  @BeanProperty
  var cMessageId: Integer = null
  @BeanProperty
  var cMessageSource: String = null
  @BeanProperty
  var cMessageType: Integer = null
  @BeanProperty
  var cReaded: Integer = null

  def this(cMessageId: Integer, cMessageSource: String, cMessageType: Integer, cReaded: Integer) {
    this()
    this.cMessageId = cMessageId
    this.cMessageSource = cMessageSource
    this.cMessageType = cMessageType
    this.cReaded = cReaded
  }
}